test/proxy.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * The talkback proxy for e2e tests. When making a request to the API during testing,
+ * it tries to use the responses it previously saved in `/test/tapes` folder. If no
+ * response is found there, it:
+ * - by default, returns 'Not found'.
+ * - if you pass `--update-tapes` as a parameter, makes an actual request, and saves the response for
```

Comment:
We would want this to run periodically so that we can ensure that the tapes don't diverge too much from the API responses.

Reply:
> Appreciate the grammar fixes to documentation.

It's all because of WebStorm complaining about them :)

Reply:
Should we set up the weekly (monthly/daily?) action for this, @dhruvkb ?